tests/test__collection_api.py

```diff
@@ -3454,11 +3454,6 @@ def test__aggregate_not_implemented(self):
                 {'$project': {'a': {'$concatArrays': [[0, 1], [2, 3]]}}},
             ])
 
-        with self.assertRaises(NotImplementedError):
-            self.db.collection.aggregate([
-                {'$project': {'a': {'$setEquals': [[2], [1, 2, 3]]}}},
```

Comment:
Please keep this one but with `setIntersection` to keep code coverage high.

Reply:
Done!